README.md

```diff
@@ -49,6 +49,30 @@ npm install --save-exact @holochain/client
   }, 30000)
 ```
 
+### Use AppAgentWebsocket
+```typescript
+  const signalCb = (signal: AppSignal) => {
+    // impl...
+    resolve()
+  }
+
+  const TIMEOUT = 12000
+  // default timeout is set to 12000
+  const appWs = await AppWebsocket.connect(`ws://localhost:${appPort}`, 12000, signalCb)
+
+  const client = new AppAgentWebsocket(appWs, 'installed_app_id')
+
+  // default timeout set here (30000) will overwrite the defaultTimeout(12000) set above
+  await client.callZome({
+   cap: null,
+   role_id: 'dnas_role_id', // role_id is unique per app, so you can unambiguously identify your dna with role_id in this client,
```

Comment:
This is now called role name right?

Reply:
Hmm, just saw that.

@jost-s - should I update `role_id` to `role_name` across this whole repo? Or leave that to you in a separate pr

Reply:
@robbiecarlton I'll either do this in signed zome calls or in a separate branch.